Assets/Scripts/WheelMotor.cs

```diff
@@ -58,6 +58,8 @@ private void UpdatePosition()
 
     private void Render()
     {
+        Quaternion swerve = transform.parent.localRotation;
+        _wheel.steerAngle = (swerve.y < 0 ? -1 : 1) * 2 * Mathf.Acos(swerve.w) * Mathf.Rad2Deg;
```

Comment:
`acos` is numerically unstable when `swerve.w` is approximately 1 or -1. A better way of calculating this is:

```C#
swerve.ToAngleAxis(out _wheel.steerAngle, out Vector3 ax);
_wheel.steerAngle *= Mathf.Sign(ax.y);
```

If you wanted to do the math yourself, you could use `atan2(magnitude(x,y,z), abs(w))` instead of `acos(w)`. In general, it's a good habit to avoid unstable functions like `acos`